### PR TITLE
Update SDK README and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by racbc**
Updated the README and CONTRIBUTING files to encourage developer contribution.

### Motivation
Want to increase developer contributions.

### Solution
Added more information on how to contribute.

### Open questions
Please offer feedback as to whether the improvements are helpful.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,56 +1,96 @@
...
+
+If your bug hasn’t been fixed, follow these steps to file a bug report:
+
+1. [Open an issue in GitHub][10].
+2. Add a title for your bug report. It should briefly describe the problem.
+3. Follow the template that appears in the Write text box. This is the best way to describe the bug.
+    * **Describe the bug** - A clear and concise description of what the bug is.
+    * **To Reproduce** - Steps to reproduce the behavior.
+    * **Expected behavior** - A clear and concise description of what you expected to happen.
+    * **Additional context** - You can include any relevant stack traces or debugging output in this section.
```
Comment (shrimalmadhur) [inline CONTRIBUTING.md:35]: This is probably redundant as we can put this in issue template

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,56 +1,96 @@
...
+
+To submit a feature request, follow these steps:
+
+1. [Open an issue in GitHub][10].
+2. Add a title for your feature request. It should briefly describe your requested feature.
+3. Follow the template that appears in the Write text box. This is the best way to explain your request. Be clear and concise in your responses.
+    * **Is your feature request related to a problem? Please describe.** - Describe what the problem is. For example, “I'm always frustrated when [...]”.
+    * **Describe the solution you'd like** - Describe what you want to happen.
+    * **Describe alternatives you've considered** - Describe any alternative solutions or features you've considered.
+    * **Additional context** - You can add any other context or screenshots about the feature request in this section.
```
Comment (shrimalmadhur) [inline CONTRIBUTING.md:50]: This is probably redundant as we can put this in issue template

File: CONTRIBUTING.md (hunk 1 above)
Comment (racbc) [reply CONTRIBUTING.md]: Removed in latest commit.

File: CONTRIBUTING.md (hunk 2 above)
Comment (racbc) [reply CONTRIBUTING.md]: Removed in latest commit.

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**